packages/mllp-server/src/utils.ts

```
@@ -17,14 +17,20 @@ import { HieConfigDictionary } from "@metriport/core/external/hl7-notification/h
 import { MetriportError } from "@metriport/shared";
 import * as Sentry from "@sentry/node";
 import IPCIDR from "ip-cidr";
+import { buildDayjs } from "@metriport/shared/common/date";
+import { HL7_FILE_EXTENSION } from "@metriport/core/util/mime";
+import { HieConfigDictionary } from "@metriport/core/external/hl7-notification/hie-config-dictionary";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'HieConfigDictionary'. Consider to delete it or rename it.

'HieConfigDictionary' is defined here:


(lint/suspicious/noRedeclare)